lib/mongo/bulk_write.rb

```diff
@@ -131,7 +131,9 @@ def base_spec(operation_id)
         :coll_name => collection.name,
         :write_concern => write_concern,
         :ordered => ordered?,
-        :operation_id => operation_id
+        :operation_id => operation_id,
+        :bypass_document_validation =>
+            options[:bypass_document_validation] || false
```

Comment:
You can just make this

```ruby
!!options[:bypass_document_validation]
```

The double bang will convert the value to a boolean. If you try this out in irb (Ruby console), you'll see how it works:

```ruby
> !!nil
 => false 
> !!false
 => false 
> !!true
 => true
```